CHANGELOG.rst

```diff
@@ -9,8 +9,18 @@ No unreleased changes.
 21.0 - 2021-07-03
 ~~~~~~~~~~~~~~~~~
 
-* `packaging` is now only compatible with Python 3.6 and above.
```

Comment:
I don't think this line should be removed?

Reply:
It's reintroduced with more granularity below:

```
* Drop support for Python 2.7, Python 3.4 and Python 3.5.
```